lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/model/MongoGroup.java

```diff
@@ -0,0 +1,50 @@
+package io.lumeer.storage.mongodb.model;
+
+import io.lumeer.api.model.Group;
+
+public class MongoGroup {
+
+   private String id;
+   private String name;
+   private String organizationId;
```

Comment:
Once the problem with the scope of `MongoGroupDao` is fixed, `organizationId` will no longer be needed here.